homeassistant/components/matter/const.py

```diff
@@ -8,3 +8,7 @@
 
 DOMAIN = "matter"
 LOGGER = logging.getLogger(__package__)
+
+# prefixes to identify device identifier id types
+ID_TYPE_DEVICE_ID = "device_id"
```

Comment:
To make it easier to match the type id and not have an underscore before the underscore that separates the type id from the rest of the identifier.
```suggestion
ID_TYPE_DEVICE_ID = "deviceid"
```

Reply:
adjusted